engine/planner.go

```diff
@@ -108,9 +109,13 @@ func (p *planner) alloc(ctx context.Context, n int) error {
 	logger := logger.FromContext(ctx)
 	logger.Debugf("allocate %d servers", n)
 
+	namePrefix := p.namePrefix
+	if namePrefix == "" {
+		namePrefix = "agent-"
+	}
```

Comment:
Is this block necessary given that we default to `agent-`?

```go
NamePrefix string `envconfig:"DRONE_AGENT_NAME_PREFIX" default:"agent-"`
```

Alternatively, would it be also necessary to have this in `server/servers.go`?

Reply:
`server/servers.go` uses the config directly which has a default so it's not needed while the planner is a separate struct. It doesn't necessarily draw from the config. At least that's my iterpretation

Reply:
Same interpretation here.